README.md

```diff
@@ -100,6 +100,22 @@ For extra security, you can disable certain Redis commands (this is especially i
       - CONFIG
       - SHUTDOWN
 
+Enable replication.
+
+    redis_replication: false
+
+A resolvable dns name or IP address of the primary redis instance.
+
+    redis_slaveof: false
+
+Replication can be further configured.
```

Comment:
As Geerlingguy provides small info about the vars in the readme, perhaps you can do the same.